demo/android/STT/.gitignore

```diff
@@ -0,0 +1,13 @@
+*.iml
+.gradle
+/local.properties
+/.idea/
+.DS_Store
+/build
+/captures
+.externalNativeBuild
+release
+porcupinedemo.jks
```

Comment:
I know you copied this from Porcupine Activity Demo, but this line down doesn't apply to this demo, so maybe remove them